src/App.js

```diff
@@ -0,0 +1,26 @@
+import './App.css';
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+
+import { fetchGreeting } from './redux/greetingSlice';
+
+function App() {
+  const dispatch = useDispatch();
+  const { isLoading, message } = useSelector((state) => state.greeting);
+
+  useEffect(() => {
+    dispatch(fetchGreeting());
+  }, [dispatch]);
+
+  if (isLoading) return (<h3>Loading</h3>);
+
+  return (
+    <div>
+      <p>
+        {`Greeting: ${message}`}
+      </p>
+    </div>
+  );
```

Comment:
- [ ] From the project guideline, the Greeting should not be implemented here. It should be implemented in its component, imported into the App component, and displayed with its route. Kindly move this implementation to a Greeting component and import it instead. The App component is intended as the main component, so other components should not be implemented here.